.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Affected Version
+
+Show version numbers by pasting the output of `composer info --direct`.
+Alternatively, hover over the SilverStripe logo in the CMS to basic version information.
```

Comment:
> hover over the SilverStripe logo in the CMS to basic version information

"get" plz :) (probably easiest for whoever merges to just manually edit afterward)

Reply:
fixed on merge